.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,61 @@
+name: Bug report
```

```suggestion
name: Bug Report
```
The blank issue is each-word-capitalized, maybe this should be too?